modules/generator/config.go

```diff
@@ -31,6 +32,8 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet)
 	cfg.Processor.RegisterFlagsAndApplyDefaults(prefix, f)
 	cfg.Registry.RegisterFlagsAndApplyDefaults(prefix, f)
 	cfg.Storage.RegisterFlagsAndApplyDefaults(prefix, f)
+	// setting default for max span age before discarding to 30 sec
+	cfg.MaxSpanAge = 30
```

Comment:
I'm curious how this default behaves in practices. I honestly have no clue what a typical latency is between the span creation and ingestion by Tempo.

Reply:
![Screen Shot 2022-08-24 at 10 38 14 AM](https://user-images.githubusercontent.com/94262131/188713938-d88a369f-e9b5-4ef4-b042-bd5b549d6c63.png)
![Screen Shot 2022-08-24 at 10 39 00 AM](https://user-images.githubusercontent.com/94262131/188713940-158f7f39-ddc1-424b-8e06-e1ec2456069c.png)
These are the ingestion latency numbers on ops for a few days. Do we think setting it at 30s is a right or is it too aggressive? @kvrhdn @joe-elliott

Reply:
30s as a default looks good to me. It seems this should be include 99% of the data while excluding 1% that is lagging behind.
It's also configurable, so other people can switch it up.